web-app/src/translations/en.tsx

```diff
@@ -515,7 +515,7 @@ export default {
 				"Learn how you can lower turnover rates, decrease recruiting costs, improve your recruiting practices, and much more with our resources",
 			resources: "Resources",
 			headerText:
-				"Recruit and retain the best workforce in Tijuna with Vize",
+				"Recruit and retain the best workforce in Tijuana with Vize",
```

Comment:
Was too lazy to create a separate branch for a single word spelling correction

Reply:
That's fine. It's a small change.

Reply:
That's fine. It's a small change.